.github/workflows/scanner-release-vuln-update.yaml

```diff
@@ -64,10 +92,16 @@ jobs:
     - name: Set up Cloud SDK
       uses: google-github-actions/setup-gcloud@v2
 
+    - uses: ./.github/actions/download-artifact-with-retry
+      with:
+        name: nvd
+        path: .
+
     - name: Update vulnerabilities
       env:
         STACKROX_NVD_API_KEY: ${{ secrets.NVD_API_KEY }}
         STACKROX_NVD_API_CALL_INTERVAL: 6s
+        STACKROX_NVD_ZIP_PATH: nvd.zip
```

Comment:
Just confirming this is expected to do nothing for 4.4.x, and will only work for 4.5+, right?